docs/guide/using_existing_contracts.rst

```diff
@@ -1,6 +1,9 @@
 Using existing contracts
 ========================
 
+Existing contracts
+------------------
+
```

Comment:
I'm not sure if it is necessary. Do you have a strong opinion on that?

Reply:
It's currently not visible on the right side.
<img width="180" alt="image" src="https://user-images.githubusercontent.com/65187002/215724921-849bc6d9-84df-4195-9653-8f0c9faebc89.png">

Reply:
Right, but the title is about that section.

Reply:
Yes, but it's nice to have every topic on the right side.
+ it's already like this in serialization section for example
<img width="385" alt="image" src="https://user-images.githubusercontent.com/65187002/215730947-5354284f-a279-49f9-ad38-cb71240cfa9c.png">

Reply:
I think it's okay to add a subsection here if you think it increases readability.

Reply:
Fine, I will not complain about it, since it makes the docs more readable 👍